.eslintrc.json

```diff
@@ -1,16 +1,11 @@
 {
   "extends": ["eslint:recommended", "plugin:import/errors", "prettier"],
-  "rules": {
-    "no-console": [
-      "error",
-      {
-        "allow": ["info", "error"]
-      }
-    ]
-  },
```

Comment:
Why is this removed?

Reply:
I see it's moved into it's own config file, right?

Reply:
That's correct -- most of the files aside from that one folder SHOULD have additional console output (for example, the main MDN-BOB.js script) because they are to be run in the command line, not the browser.